samples/tab-request-approval/nodejs/Readme.md

```diff
@@ -0,0 +1,133 @@
+---
+page_type: sample
+description: This sample shows a feature where requester can request for any task approval from manager and manager can see the pending request by user on the click of activity feed notification and can approve or reject the request.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- nodejs
+extensions:
+contentType: samples
+createdDate: "26-11-2021 14:00:00"
+---
+
+# Tab request approval with activity feed notification.
+
+This sample shows a feature where:
+1. Requester : Can request for any task approval from manager by sending activity feed notification and can see his request status.
+2. Manager : Can see the pending approval request raised by user on the click of activity feed notification and can approve or reject the request.
+
+User Persona:
+
+- Send request to the manger for task approval.
+
+![Request from user](Images/TaskRequest.png)
+
+- Request status
+
+![Request status](Images/RequestStatus.png)
+
+Manager Persona:
+
+- Activity feed notification of approval request.
+
+![Notification](Images/RequestNotification.png)
```

Comment:
Images are not aligned properly here:
![image](https://user-images.githubusercontent.com/59052547/143542006-8136fb8f-25b5-44b6-b2e8-ca2d15aa47b0.png)